src/CatalogApplication/Models/Tags.cs

```diff
@@ -2,6 +2,12 @@ namespace CatalogApplication.Models
 {
     public class CatalogData
```

Comment:
nit: your naming convension. You should have a more descriptive name. As someone who is looking at your code for the first time, I don't know what CatalogData means. I know what a tag is and I know what a rating is, but i don't know what `CatalogData` means